import-export-cli/git/gitUtils.go

```diff
@@ -700,7 +704,9 @@ func logChangedFiles(changedFileList []string) {
 }
 
 // subPath denotes a single changed file retrieved from "git diff" command. This path is scanned and then identifies the
-//  project type (API, App,.. ) from this method.
+//
+//	project type (API, App,.. ) from this method.
+//
```

Comment:
Seems this change is not required. Please check.

Reply:
Resolved by https://github.com/wso2/product-apim-tooling/pull/1027/commits/bf810875b56eb0bb7f3328899408af14d8d243e8